demo/flutter/lib/main.dart

```diff
@@ -53,18 +62,24 @@ class _MyAppState extends State<MyApp> with WidgetsBindingObserver {
 
   @override
   void didChangeAppLifecycleState(AppLifecycleState state) async {
-    if (state == AppLifecycleState.inactive) {
+    if (state == AppLifecycleState.paused) {
```

Comment:
I seem to remember there was a specific reason I chose inactive over paused. Make sure you test that putting it in the background of both platforms doesn't mess anything up

Reply:
I remember this messes up the program when mic permission pop up. Right now seems ok going to background and returning back.